static/src/javascripts/projects/common/modules/identity/upsell/account-creation/AccountCreationForm.js

```diff
@@ -20,41 +19,43 @@ class AccountCreationForm extends Component {
             errors: [],
         });
 
-        reqwest({
-            url: '/password/guest',
-            method: 'post',
-            data: {
-                csrfToken: this.props.csrfToken,
-                token: this.props.accountToken,
-                password: this.state.password,
-            },
-            success: () => {
-                ophan.record({
-                    component: 'set-password',
-                    value: 'set-password',
+        fetch(
+            '/password/guest',
+            {
+                method: 'POST',
+                body: JSON.stringify({
+                    csrfToken: this.props.csrfToken,
+                    token: this.props.accountToken,
+                    password: this.state.password,
+                })
+            }
+        )
+        .then( () => {
+            ophan.record({
+                component: 'set-password',
+                value: 'set-password',
+            });
+            this.props.onAccountCreated();
+        })
+        .catch( (response) => {
+            reportError(
+                Error(response),
+                {
+                    feature: 'identity-create-account-upsell',
+                },
+                false
+            );
+            try {
+                const apiError = JSON.parse(response.responseText)[0];
+                this.setState({
+                    errors: [apiError.description],
                 });
-                this.props.onAccountCreated();
-            },
-            error: response => {
-                reportError(
-                    Error(response),
-                    {
-                        feature: 'identity-create-account-upsell',
-                    },
-                    false
-                );
-                try {
-                    const apiError = JSON.parse(response.responseText)[0];
-                    this.setState({
-                        errors: [apiError.description],
-                    });
-                } catch (exception) {
-                    this.setState({ errors: [genericErrorStr] });
-                }
-            },
-            complete: () => {
-                this.setState({ isLoading: false });
-            },
+            } catch (exception) {
+                this.setState({ errors: [genericErrorStr] });
+            }
+        })
+        .finally( () => {
+            this.setState({ isLoading: false });
```

Comment:
This is the biggest change of interface going from `reqwest` to `fetch`.

I think the following mappings are correct, but will require further testing:
- `success` &rarr; `.then()`
- `error: response` &rarr; `.catch(response)`
- `commplete` &rarr; `.finally()`

cc. @guardian/identity